README.md

```diff
@@ -1,20 +1,14 @@
 [![Main and pull request checks](https://github.com/Aiven-Open/commons-for-apache-kafka-connect/actions/workflows/main_push_workflow.yml/badge.svg)](https://github.com/Aiven-Open/commons-for-apache-kafka-connect/actions/workflows/main_push_workflow.yml)
 
-# Aiven's Common Module for Apache Kafka® connectors
+# Aiven's Apache Kafka Connectors
 
-Shared common functionality among Aiven's connectors for Apache Kafka:
-- [Aiven GCS Connector](https://github.com/aiven-open/gcs-connector-for-apache-kafka)
-- [Aiven S3 Connector](https://github.com/aiven-open/s3-connector-for-apache-kafka)
-
-# Usage
-
-When installing this library on Kafka Connect, use a specific plugin path, and **avoid placing them on the same path as the Kafka Connect binaries**, as some libraries may have conflicting versions.
+- [Aiven Connector Commons](../commons/README.md)
+- [Aiven GCS Connector](../gcs-connector/README.md)
+- [Aiven S3 Connector](../s3-connector/README.md)
```

Comment:
These links are broken, this is top level README compared to commons/README where `../` is valid prefix.